### PR TITLE
Refactored broadcast page

### PR DESCRIPTION
**Pull request by FireQueen-3010**
## Issue that this pull request solves

 For : #490 (Broadcast section)

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [X] Other (please describe): Refactored Broadcasts page

## Checklist

_Put an `x` in the boxes that apply_

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by FireQueen-3010** [comment]
@Kajol-Kumari Please review and let me know regarding the pr!

File: frontend/src/pages/Broadcast/Component/Carousel/Modal/modals.module.scss [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+  border-radius: 10px;
 }
 
 .modal {
   display: flex;
   align-items: center;
   justify-content: center;
+  align-items: center;
+}
+.link {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Broadcast/Component/Carousel/Modal/modals.module.scss:13]: please leave one blank line space here

File: frontend/src/pages/Broadcast/Component/Carousel/Modal/modals.module.scss [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+  background-color: #016795;
+  font-size: 25px;
+  text-align: center;
+  color: white;
+  font-weight: bold;
+  padding: 1em 0;
+}
+.link:hover {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Broadcast/Component/Carousel/Modal/modals.module.scss:21]: same here. One line space

File: frontend/src/pages/Broadcast/Component/Carousel/Modal/modals.module.scss [hunk 3]
```diff
@@ -16,6 +30,15 @@
   color: white;
   font-weight: bold;
   padding: 1em 0;
+  align-items: center;
+}
+.close-btn {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Broadcast/Component/Carousel/Modal/modals.module.scss:35]: same here. One line space

File: frontend/src/pages/Broadcast/Component/Carousel/Modal/modals.module.scss [hunk 4]
```diff
@@ -16,6 +30,15 @@
...
+  float: right;
+  padding-right: 25px;
+}
+.close-btn:hover {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Broadcast/Component/Carousel/Modal/modals.module.scss:39]: same here. One line space

File: frontend/src/pages/Broadcast/Component/Carousel/carousel.module.scss [hunk 5]
```diff
@@ -23,6 +24,22 @@
   padding: 1em;
 }
 
+.card-head {
+  font-size: 25px;
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Broadcast/Component/Carousel/carousel.module.scss:28]: let's keep it `24px`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@FireQueen-3010 Please do the following changes-
1. Reduce the word limit from 600 to 420 or 450 [here](https://github.com/HITK-TECH-Community/Community-Website/blob/main/frontend/src/pages/Broadcast/Component/Carousel/Carousel.jsx#L73)
2. The View modal for large screens(not XL) get's hidden a bit, Can you please fix that as well? I will merge this under `level-3` then :)

Large screen view Broadcast Bug :arrow_down: 
Screenshot from 2021-03-18 18-57-28

**Comment by FireQueen-3010** [comment]
Ok @Kajol-Kumari

File: frontend/src/pages/Broadcast/Component/Carousel/Modal/modals.module.scss (hunk 2 above)
Comment (FireQueen-3010) [reply frontend/src/pages/Broadcast/Component/Carousel/Modal/modals.module.scss:21]: before .link:hover right?

**Comment by FireQueen-3010** [comment]
Done @Kajol-Kumari  with all of that :)

**Comment by Kajol-Kumari** [comment]
Hey @FireQueen-3010 my screen resolution is `1366x768` and I am still facing that large screen bug. Can you please verify and fix it?

**Comment by FireQueen-3010** [comment]
Okay

**Review by Kajol-Kumari — APPROVED** [review]
Hey @FireQueen-3010 Thanks for incorporating all the suggested changes
LGTM :+1: 

Just a note -  From next time don't use `git add .` instead use `git add <file_name>` so that you don't add `package-lock.json` file in the commit :)

**Comment by FireQueen-3010** [comment]
Yeah sure I'll keep that in mind the next time @Kajol-Kumari . Thank you so much! 😄

**Pull request merged**